src/client/pages/OfferNew/Tabs/Tabs.tsx

```diff
@@ -1,26 +1,38 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import { AnimateSharedLayout } from 'framer-motion'
 import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
-import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
 import { Tab } from './Tab'
 
 const TabList = styled.div`
   display: flex;
-  margin: 1rem 0 1.25rem 0;
-  box-shadow: 0px 2px 0px ${colorsV3.gray300};
-  overflow-x: scroll;
+  overflow-x: auto;
   scrollbar-width: none; /* Firefox */
   -ms-overflow-style: none; /* Internet Explorer 10+ */
+
   &::-webkit-scrollbar {
     width: 0;
     height: 0;
   }
-  ${MEDIUM_SCREEN_MEDIA_QUERY} {
-    margin: 1rem 1rem 2.5rem 0;
+`
+
+const TabContainer = styled.div`
+  position: relative;
+  margin-bottom: 1.25rem;
+  ${LARGE_SCREEN_MEDIA_QUERY} {
```

Comment:
Maybe add a space here

```suggestion
  margin-bottom: 1.25rem;

  ${LARGE_SCREEN_MEDIA_QUERY} {
```